content/pages/about.md

```diff
@@ -54,7 +54,7 @@ workshop "Digital Research in Old Norse Studies: Data Management and Infrastruct
 
 Emily Lethbridge
 
-[Icelandic Saga Map](http://sagamap.hi.is/is/) / Stofnun Árna Magnússonar í íslenskum fræðum, Reykjavík
+Emily works on medieval Icelandic manuscripts and literary culture, landscape and place-names. Digital projects: [Icelandic Saga Map](http://sagamap.hi.is/is/) and [Nafnið.is](https://nafnið.is/)
```

Comment:
so far we always had one line name, one line affiliation/infrastructures and then a paragraph bio. I don't mind giving up that schema... but I'm a bit a sucker for consistency. ;-)

Reply:
Fair. Changed accordingly